drivers/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # Define mappings for drivers
 set(DRIVER_MAPPINGS
     chimera-convolve:cluster
-    chimera-open:cluster
+    chimera-open:cluster,uart_${UART_DRIVER_VERSION}
```

Comment:
Please see the comment below. I am not a big fan of the `uart_${UART_DRIVER_VERSION}` and suggest `uart_opentitan`.